src/printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print("x is %(x)i, y is %(y).2f, z is %(z)s" % {'x': x, 'y': y, 'z': z})
```

Comment:
Mine looks like: print("x is %d, y is %0.2f, z is '%s'"% (x, y, z))

Reply:
Personally I'm not a fan of positional arguments. Afraid of a typo that would put something out of place.